File: Praktikum_ALPRO/Pertemuan-3-Looping/Latihan 1/main_app.py
# Menampilkan Bilangan Genap dari 1 - 50

print("Angka genap antara 1-50 adalah: ")
for i in range(1, 51): # saya lebihkan menjadi 51 karena untuk stop di 50
    if i%2 == 0:
        print(i)

File: Praktikum_ALPRO/Pertemuan 3 Looping/Latihan 1/main_app.c
// MEnampilkan Bilangan Genap dari 1-50

#include <stdio.h>

int main(){
    printf("Angka genap antara 1-50 adalah: \n");
    for (int i=1;i<=50;i++){
        if (i%2==0){
            printf("%d\n",i);
        }
    }
    return 0;
}